docs/src/pages/components/data-grid/columns/columns.md

```diff
@@ -127,7 +127,7 @@ By default, each column header displays a column menu. The column menu allows ac
 
 ## Column selector
 
-To enable the column selector in the toolbar you need to add the `showToolbar` prop to the data grid.
+To enable the column selector in the toolbar you need to add the `Header: GridHeader` the `components` grid prop.
```

Comment:
These instruction don't seem to match the demo that follows. If I understand correctly, it should read:

```suggestion
To enable the the toolbar you need to add `Toolbar: GridToolbar` to the grid `components` prop.
```

Do we need to document this for each of the toolbar functions?

Reply:
Thank you. I tried to replace it where I could find it. I think they should be mentioned in the places where the respective feature docs are.